### DISCUSSION TITLE
Step-by-step guide section 3 misses some detail

### DISCUSSION THREAD

**Discussion by gyroscopes** [dojo-engineering/feedback#6] [Docs]
In the step by step guide here
https://docs.dojo.tech/docs/accept-payments/checkout-page/step-by-step-guide

Section 3  is entitled "Handle post-payment events"
The title is a little confusing as it is really about setting up where the post-payment events will go. e.g. setting up the webhooks but not talking about getting/processing the webhooks.

This doesn't mention that this step can be done once in the developer portal.

Maybe /cs/verify-webhooks.cs would be better shown in step 3
rather /cs/enable-webhooks.cs

**Comment by RomanKondratyev** [comment]
Good point - we could extend the section to include two things required:

- Webhooks setup
- Webhooks events handling

@monica-m-ps

**Comment by monica-m-ps** [comment]
@gyroscopes Thanks for your feedback. I'm working on this, and shall keep you updated when resolved.
